src/shared/components/action-buttons/ShareDropdown.tsx

```diff
@@ -61,6 +61,13 @@ const CopyLinkWebsite = ({
   );
 };
 
+const visibleElement = (onClick: () => unknown) => (
```

Comment:
Example of where the visibleElement has been put outside the component because it's static